tool_commands/raster_checker/test_raster_checker.py

```diff
@@ -27,30 +34,28 @@ def setUp(self):
         self.metadata = MetaData(bind=engine)
         self.datamodel = DataModelSource(self.metadata)
         self.rc_entrees = RasterCheckerEntrees(self.datamodel, session)
```

Comment:
Ongerelateerd aan dit pull request... Dit zie ik nu pas: wat wordt er met "entrees" bedoeld? In het Engels is dat "voorgerecht".
Moet dat "entries" zijn?

Reply:
oi oi oi, moet idd entries zijn
ik heb alle entrees --> entries, en entree --> entry